### PR TITLE
[Docs] Translate kubernetes-containerd.md to zh.

### LINKED ISSUE #1099: [Doc] Translate kubernetes-containerd.md to zh
### The MD file link you want to translate (must be in English) 
https://github.com/WasmEdge/WasmEdge/blob/master/docs/book/en/src/kubernetes/kubernetes/kubernetes-containerd.md

### Target language
zh: https://github.com/WasmEdge/WasmEdge/blob/master/docs/book/zh/src/kubernetes/kubernetes/kubernetes-containerd.md

### PR DESCRIPTION
**Pull request by Bevisy**
* [Docs] Translate kubernetes-containerd.md to zh.

Fix #1099

Signed-off-by: Binbin Zhang <binbin36520@gmail.com>

### CODE REVIEW

**Comment by Bevisy** [comment]
PTAL @alabulei1

File: docs/book/zh/src/kubernetes/kubernetes/kubernetes-containerd.md [hunk 1]
```diff
@@ -1 +1,119 @@
...
+
+## 快速开始
+
+[GitHub 仓库](https://github.com/second-state/wasmedge-containers-examples/) 包含在 Kubernetes + containerd 上运行我们的示例应用程序的脚本和 Github Actions。
+
+* 简单的 WebAssembly 示例 [快速开始](https://github.com/second-state/wasmedge-containers-examples/blob/main/kubernetes_containerd/README.md) | [Github Actions](https://github.com/second-state/wasmedge-containers-examples/blob/main/.github/workflows/kubernetes-containerd.yml) | [成功运行](https://github.com/second-state/wasmedge-containers-examples/runs/4577789181?check_suite_focus=true#step:6:3010)
+* 基于 WebAssembly 的 HTTP 服务 [快速开始](https://github.com/second-state/wasmedge-containers-examples/blob/main/kubernetes_containerd/http_server/README.md) | [Github Actions](https://github.com/second-state/wasmedge-containers-examples/blob/main/.github/workflows/kubernetes-containerd-server.yml) | [成功运行](https://github.com/second-state/wasmedge-containers-examples/runs/4577789184?check_suite_focus=true#step:6:3016)
+
+在本节的其余部分，我们将详细解释这些步骤。
+我们将假设你已经 [安装并配置了containerd](../cri/containerd.md)，以便与 WasmEdge 容器镜像一起工作。
```
Comment (alabulei1) [inline docs/book/zh/src/kubernetes/kubernetes/kubernetes-containerd.md:11]: 10和11 是一行

File: docs/book/zh/src/kubernetes/kubernetes/kubernetes-containerd.md [hunk 2]
```diff
@@ -1 +1,119 @@
...
+
+## 安装和启动 Kubernetes
+
+从终端窗口运行以下命令。
```
Comment (alabulei1) [inline docs/book/zh/src/kubernetes/kubernetes/kubernetes-containerd.md:15]: 15 和 16 也是一行。从终端窗口运行以下命令，为本地开发设置了 Kubernetes。

File: docs/book/zh/src/kubernetes/kubernetes/kubernetes-containerd.md [hunk 3]
```diff
@@ -1 +1,119 @@
...
+### 一个基于 WebAssembly 的 HTTP 服务
+
+[另一篇文章](../demo/server.md)解释了如何编译、打包和发布一个简单的 WebAssembly HTTP 服务应用程序作为容器镜像到 Docker hub。
+在 Kubernetes 集群中运行 Docker Hub 中基于 WebAssembly 的镜像，方法如下。
+
+```bash
+sudo cluster/kubectl.sh run --restart=Never http-server --image=avengermojo/http_server:with-wasm-annotation --annotations="module.wasm.image/variant=compat" --overrides='{"kind":"Pod", "apiVersion":"v1", "spec": {"hostNetwork": true}}'
+```
+
+由于我们在`kubectl run`命令中使用了`hostNetwork`，HTTP 服务器镜像运行在本地网络上，IP 地址是`127.0.0.1`。
```
Comment (alabulei1) [inline docs/book/zh/src/kubernetes/kubernetes/kubernetes-containerd.md:111]: 代码块前后空一格。

File: docs/book/zh/src/kubernetes/kubernetes/kubernetes-containerd.md [hunk 4]
```diff
@@ -1 +1,119 @@
...
+现在，你可以使用`curl`命令来访问 HTTP 服务。
```
Comment (alabulei1) [inline docs/book/zh/src/kubernetes/kubernetes/kubernetes-containerd.md:112]: 代码块前后空一格

**Review by alabulei1 — COMMENTED** [review]
Thanks! please check out my feedback!

File: docs/book/zh/src/kubernetes/kubernetes/kubernetes-containerd.md (hunk 1 above)
Comment (Bevisy) [reply docs/book/zh/src/kubernetes/kubernetes/kubernetes-containerd.md]: Thanks, fixed now

**Comment by Bevisy** [comment]
> 

Thanks for your review, all the problems pointed out above have been fixed. PTAL

File: docs/book/zh/src/kubernetes/kubernetes/kubernetes-containerd.md [hunk 5]
```diff
@@ -1 +1,117 @@
 # Kubernetes + containerd
+
+## 快速开始
+
+[GitHub 仓库](https://github.com/second-state/wasmedge-containers-examples/) 包含在 Kubernetes + containerd 上运行我们的示例应用程序的脚本和 Github Actions。
+
+* 简单的 WebAssembly 示例 [快速开始](https://github.com/second-state/wasmedge-containers-examples/blob/main/kubernetes_containerd/README.md) | [Github Actions](https://github.com/second-state/wasmedge-containers-examples/blob/main/.github/workflows/kubernetes-containerd.yml) | [成功运行](https://github.com/second-state/wasmedge-containers-examples/runs/4577789181?check_suite_focus=true#step:6:3010)
+* 基于 WebAssembly 的 HTTP 服务 [快速开始](https://github.com/second-state/wasmedge-containers-examples/blob/main/kubernetes_containerd/http_server/README.md) | [Github Actions](https://github.com/second-state/wasmedge-containers-examples/blob/main/.github/workflows/kubernetes-containerd-server.yml) | [成功运行](https://github.com/second-state/wasmedge-containers-examples/runs/4577789184?check_suite_focus=true#step:6:3016)
+
+在本节的其余部分，我们将详细解释这些步骤。我们将假设你已经 [安装并配置了containerd](../cri/containerd.md)，以便与 WasmEdge 容器镜像一起工作。
```
Comment (hydai) [inline docs/book/zh/src/kubernetes/kubernetes/kubernetes-containerd.md:10]: `已经 [安装并配置了containerd]` -> `已经[安装并配置了 containerd]`

**Review by hydai — CHANGES_REQUESTED** [review]

File: docs/book/zh/src/kubernetes/kubernetes/kubernetes-containerd.md (hunk 5 above)
Comment (Bevisy) [reply docs/book/zh/src/kubernetes/kubernetes/kubernetes-containerd.md]: Fixed. Thanks

**Comment by alabulei1** [comment]
@hydai  I think it's ready to merge this PR.

**Review by hydai — APPROVED** [review]

**Pull request merged**